### ISSUE TITLE
Find hosted SQLite DB for Next project

### ISSUE THREAD

**Issue by oliverjam** [foundersandcoders/coursebook#723]
The Next.js projects are deployed to Vercel. This is a serverless platform and so does not support any DBs—data persistence must be hosted somewhere else. For ease we have recommended ElephantSQL previously (really simple free hosted Postgres). Now that we're switching to SQLite it would be a big jump for them to suddenly have to learn PG, especially on top of Next. Ideally we can find a hosted SQLite solution or something (I have no idea how this would even work since SQLite is an in-process DB, but I'll do some research).

**Comment by oliverjam** [comment]
Actually deploying Next.js to Heroku is [super simple](https://github.com/mars/heroku-nextjs#production-deployment) so maybe we shouldn't reinvent the wheel and just keep using a regular host instead of jumping into a weird serverless env.

**Comment by oliverjam** [comment]
We will need to update the project guide to recommend Heroku instead of Vercel, and link to https://github.com/mars/heroku-nextjs#production-deployment

**Comment by Albadylic** [comment]
Is there something we can recommend for final projects? Heroku's gonna peace out in November, right? Which would be just before the end of the programme and likely to be a spanner in the works for TFB

**Comment by oliverjam** [comment]
Yeah for a more serious project they'll probably need to use something that requires a credit card unfortunately (assuming they stick to the stack from the course). E.g. fly.io or render.com with an attached storage volume.

If they end up doing what most teams have done and handing off their DB/backend to a SaaS provider like Supabase then it won't matter much, they can deploy to Vercel or whomever.
